.eslintrc

```diff
@@ -32,5 +32,30 @@
       }
     ],
     "jsx-a11y/alt-text": "warn"
-  }
+  },
+  "overrides": [
```

Comment:
Linting rules for the `tests/e2e/` files. Without these, there are ESLint errors from using the `page` global, etc...